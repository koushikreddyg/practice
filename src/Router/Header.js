import React from 'react';
import {NavLink} from 'react-router-dom';
import { Button } from 'antd';
import 'antd/dist/antd.css';

 const Header=()=>(
  <div>
 <Button offset={6} type="primary" icon="cloud-download">
  <NavLink to="/" exact={true}>Home</NavLink>
 </Button>
  </div>
)
export default Header;