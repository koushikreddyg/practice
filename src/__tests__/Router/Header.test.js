import React from 'react';
//import ReactShallowRender from 'react-test-renderer/shallow';
import {shallow} from 'enzyme';
import Header from '../../Router/Header';
test('render the header component',()=>{
    //const renderer=new ReactShallowRender();
    //renderer.render(<Header/>);
    //expect(renderer.getRenderOutput()).toMatchSnapshot();
    const wrapper=shallow(<Header/>)
    expect(wrapper).toMatchSnapshot();
})