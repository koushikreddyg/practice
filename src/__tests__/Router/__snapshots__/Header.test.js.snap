// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the header component 1`] = `
<div>
  <Button
    ghost={false}
    icon="cloud-download"
    loading={false}
    offset={6}
    prefixCls="ant-btn"
    type="primary"
  >
    <NavLink
      activeClassName="active"
      ariaCurrent="true"
      exact={true}
      to="/"
    >
      Home
    </NavLink>
  </Button>
</div>
`;
